app/models/firm.rb

```diff
@@ -173,6 +174,10 @@ def primary_advice_method
     infer_primary_advice_method
   end
 
+  def main_office
+    offices.first
```

Comment:
The definition of ```main office``` should be more than just the first office in the list. If that's the case then there's no need in even creating a method for specifying it. I hope it makes sense (it does in my head).